src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -243,20 +242,8 @@ def validatePhEDExSubscription(arguments):
     Validate all the PhEDEx arguments provided during request
     creation and assignment.
     """
-    for site in arguments.get("AutoApproveSubscriptionSites", []):
-        if site.endswith('_MSS'):
-            raise WMSpecFactoryException("Auto-approval to MSS endpoint is not allowed: %s" % site)
-    if arguments.get("SubscriptionPriority", "Low").lower() not in PhEDEx_VALID_SUBSCRIPTION_PRIORITIES:
+    if arguments.get("SubscriptionPriority", "Low").lower() not in RUCIO_RULES_PRIORITY:
```

Comment:
this line can be removed, since we already validate it in the StdBase super class.
Actually, can you please review if this method `validatePhEDExSubscription` is actually used/needed somewhere else? I am inclined to say that it can be completely removed.

Reply:
Yes it can be completely removed, I did it with https://github.com/dmwm/WMCore/pull/11181/commits/737ac4de6bef1a3d49e33f70ec16f74955cbdb5f